### PR TITLE
Clean up registry dialogs

### PR DESCRIPTION
**Pull request by maffoo**
Use paper elements to give the registry editor dialogs a common style and fix up their behavior. Also pull a few styles out of main.css and into elements themselves.

### CODE REVIEW

**Comment by joshm-3b6b58** [comment]
The menu buttons don't work for me anymore. For example, when I click "new folder" the dialog doesn't pop up.
`[labrad-registry::_createEventHandler]: listener method `newFolderClicked` not defined`

**Comment by joshm-3b6b58** [comment]
I get these scrollbars on the registry entry table now, that I don't think do anything:
screenshot from 2015-09-01 16 34 26

**Comment by maffoo** [comment]
`newFolderClicked` definitely exists; did you clean and rebuild everything? As for the scrollbars; I'll see if I can fix that; which browser are you on?

**Comment by maffoo** [comment]
oops; I had a syntax error in there that was breaking the typescript compiler. Should work now.

**Comment by joshm-3b6b58** [comment]
Scrollbars are gone and the buttons work. I really like what you did here.

File: client-js/app/elements/labrad-registry.html [hunk 1]
```diff
@@ -102,87 +102,74 @@
...
 
-    <!-- These are the dialogs -->
-    <paper-dialog id="doRename">
-      <p>Rename "<span>{{selItem}}</span>" to something else</p>
+    <paper-dialog id="newKeyDialog" modal>
+      <h1>Create new key</h1>
       <form>
-        <input></input>
+        <paper-input id="newKeyInput" label="Key"></paper-input>
+        <paper-textarea id="newValueInput" label="Value"></paper-textarea>
```
Comment (joshm-3b6b58) [inline client-js/app/elements/labrad-registry.html:126]: Textarea is a nice multiline input but if I try and input something with newlines in between input, it fails in some unexpected way. As far as I can tell it never even sends the RPC command. 

Sample input:

```
"123123123

123123123"
```

File: client-js/app/elements/labrad-registry.ts [hunk 2]
```diff
@@ -91,43 +91,129 @@ export class LabradRegistry extends polymer.Base {
...
 
-  /////////////////////
-  //Interface Functions
-  ////////////////////
-  doDelete() {
-    var self = this;
+  onKeyPress(event) {
+    console.log('onKeyPress', event);
+    switch (event.key) {
+      case 'j':
```
Comment (joshm-3b6b58) [inline client-js/app/elements/labrad-registry.ts]: What's going on here? Are these hotkeys?

Comment (maffoo) [inline client-js/app/elements/labrad-registry.ts]: oops, that was an experiment; removed for now.

File: client-js/app/elements/labrad-registry.html (hunk 1 above)
Comment (maffoo) [inline client-js/app/elements/labrad-registry.html:126]: The error message is not great, but there is an error in the console. The problem is that's not valid input; you have to escape newlines in string literals: `"123123123\n123123123"`. There's no equivalent to python's triple-quoted strings, though we could add that. Newlines do work fine between, say, list elements:

```
[1, 2, 3,
 4, 5, 6
]
```

Comment (joshm-3b6b58) [inline client-js/app/elements/labrad-registry.html:126]: If I remember correctly the old delphi registry editor would basically supress or revert newlines it didn't like. Is this behavior we want to add here eventually?

**Comment by joshm-3b6b58** [comment]
LGTM

**Pull request merged**